README.openshift.md

```diff
@@ -1,62 +1,57 @@
 ## Changing API
 
-OpenShift is split into three major repositories:
+OpenShift Kubernetes components are split into four major repositories:
 
 1. https://github.com/openshift/api/ - which holds all the external API objects definitions.
-1. https://github.com/openshift/client-go/ - which holds all the client code (written in Go).
-1. https://github.com/openshift/origin/ - which holds the actual code behind OpenShift.
+1. https://github.com/openshift/client-go/ - which holds all the client code (written in Go)
+1. https://github.com/openshift/library-go/ - which holds all the client code (written in Go).
+1. https://github.com/openshift/kubernetes/ - which holds the actual code behind OpenShift.
```

Comment:
I still think this block is strange. What are external API objects?

"which holds the actual code behind OpenShift." does not really make sense. Correct: "o/k – the core Kubernetes components with OpenShift specific patches".

Reply:
Done